gravitee-apim-console-webui/src/management/settings/custom-user-fields/migrated/dialog/custom-user-fields-dialog.component.html

```diff
@@ -0,0 +1,75 @@
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="dialog-title">
+  <span mat-dialog-title>{{ customUserFieldsDialogData.action }} user field</span>
+</div>
+
+<form [formGroup]="form" (ngSubmit)="save()">
+  <mat-dialog-content class="form-fields">
+    <mat-form-field appearance="outline" class="form-field">
+      <input
+        [matAutocomplete]="auto"
+        id="key"
+        type="text"
+        matInput
+        formControlName="key"
+        required="true"
+        data-testid="key-input"
+        (click)="initOptions()"
+      />
+      <mat-label>Key</mat-label>
+      <mat-autocomplete #auto="matAutocomplete">
+        @for (option of filteredOptions | async; track option) {
+          <mat-option [value]="option">
+            {{ option }}
+          </mat-option>
+        }
+      </mat-autocomplete>
+      <mat-error *ngIf="form.get('key').hasError('pattern')">Only a-zA-Z0-9_- characters allowed</mat-error>
+      <mat-error *ngIf="form.get('key').hasError('required')">Key is required.</mat-error>
+      <mat-error *ngIf="form.get('key').hasError('maxlength')">Key can not exceed 50 characters.</mat-error>
+      <mat-error *ngIf="form.get('key').hasError('minlength')">Key has to be at least 1 characters long. </mat-error>
+    </mat-form-field>
+
+    <mat-form-field appearance="outline" class="form-field">
+      <input id="label" type="text" matInput formControlName="label" required="true" data-testid="label-input" />
+      <mat-label>Label</mat-label>
+      <mat-error *ngIf="form.get('label').hasError('required')">Label is required.</mat-error>
+      <mat-error *ngIf="form.get('label').hasError('maxlength')">Label length can not exceed 50 characters.</mat-error>
+      <mat-error *ngIf="form.get('label').hasError('minlength')">Label has to be at least 1 characters long. </mat-error>
+    </mat-form-field>
+
+    <gio-form-slide-toggle>
+      <gio-form-label>Required</gio-form-label>
+      <mat-slide-toggle gioFormSlideToggle formControlName="required" aria-label="Required" name="required"></mat-slide-toggle>
+    </gio-form-slide-toggle>
+
+    <mat-form-field appearance="outline">
+      <mat-label>Values</mat-label>
+      <gio-form-tags-input placeholder="Type value and confirm with enter." formControlName="values"></gio-form-tags-input>
+    </mat-form-field>
+  </mat-dialog-content>
+
+  <mat-dialog-actions class="dialog-actions">
```

Comment:
```suggestion
  <mat-dialog-actions align="end">
```
what do you think of using this proposed by material directly and deleting the custom CSS added here ?

Reply:
IMHO, align="end" should align button with the red line 
![image](https://github.com/user-attachments/assets/6837b8f6-c578-439c-9db7-16ed0133d7bc)

Reply:
aligned wiht css it looks better:
![image](https://github.com/user-attachments/assets/876972f8-7a5b-4536-b43c-25b970d34b9e)

Reply:
This is one of the reasons why we use material, to prevent dev personnel preference from becoming visible. And so that the application has a global coherence.
I know this isn't already the case for this topic. and it should be a global configuration added in ui-particles. but not case by case like here.

Overall coherence should always be a priority (and even on figma) on this kind of subject IMHO
Otherwise, there's no point in sharing a designated system if nobody uses it correctly 🤷‍♂️

Reply:
so align="end" it is